internal/api/ui/login/change_password_handler.go

```diff
@@ -40,9 +40,15 @@ func (l *Login) renderChangePassword(w http.ResponseWriter, r *http.Request, aut
 		errType, errMessage = l.getErrorMessage(r, err)
 	}
 	translator := l.getTranslator(r.Context(), authReq)
+	step, ok := authReq.PossibleSteps[0].(*domain.ChangePasswordStep)
```

Comment:
Is it guaranteed that `len(authReq.PossibleSteps) > 0` is always true? Otherwise we risk an out-of-bound panic here.

Reply:
you're right, now checked